pkgs/applications/misc/trenchbroom/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, lib, git, wxGTK31, cmake, mesa, mesa_glu, freeimage, freetype, pandoc, libX11 }:
+
+stdenv.mkDerivation rec {
+  pname = "TrenchBroom";
+  version = "2019.6";
+
+  src = fetchgit {
+    url = "https://github.com/kduske/TrenchBroom.git";
+    deepClone = true;
+    rev = "v${version}";
+    sha256 = "1yx09jcz7psg4l9m89amhda30lrclzk74rfp8avbhkz9bypkqn95";
+  };
+
+  buildInputs = [ mesa mesa_glu freeimage freetype pandoc wxGTK31 libX11 ];
+  nativeBuildInputs = [ cmake git ];
```

```suggestion
  nativeBuildInputs = [ cmake ];
```
If git is fetching some info about the repo please patch that.